src/utils/config_utils.ts

```diff
@@ -22,4 +22,25 @@ export class ConfigUtils {
 		});
 		return repoPath || "";
 	}
+
+	getRepoIdByPath = (repoPath: string): number | null => {
+		if (!this.isConfigValid()) return null;
+		const path: string | undefined = Object.keys(this.config.repos).find(path => {
```

Comment:
This is fine but I think we can directly check key in the repoConfig as (avoiding find loop)
```
const repoConfig = this.config.repos[repoPath];
if (!repoConfig) return null;
return repoConfig.id;
```

Reply:
Good idea, updating.